examples/custom-animation/running-dashed-message-flow/index.js

```diff
@@ -0,0 +1,14 @@
+// Main BPMN Container
+const bpmnContainerElt = window.document.getElementById('bpmn-container');
+const bpmnVisualization = new bpmnvisu.BpmnVisualization(bpmnContainerElt);
+
+bpmnVisualization.load(getIncidentManagementBpmnDiagram(), {fit: {type: bpmnvisu.FitType.Center}});
```

Comment:
❓ can we add a margin to avoid having the pool border on top of the bpmn container border? 5 or 10px should be enough, and no impact on the preview image which doesn't display the whole diagram